app/src/main/java/practicum/getfitla_v3/Nutrition_New_Page.java

```diff
@@ -0,0 +1,38 @@
+package practicum.getfitla_v3;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+/**
+ * Created by Work on 3/17/18.
```

Comment:
I think it would be nice to have a description of what the class does at the top of each java file. It doesn't have to be long, but the documentation is important to keeping everything labeled.